toDoApp/src/screens/NewTask/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import { Pressable, SafeAreaView, Text, TextInput, View } from 'react-native'
+import { useDispatch } from 'react-redux'
+import { routes } from '../../navigation/routes'
+import { addTodo } from '../../store/todo'
+import { Color } from '../../styles/Pallete'
+import { constants } from './strings'
+import { styles } from './styles'
+import { newTaskProps } from './types'
+
+var newId = 0
```

Comment:
😂  pretty clever.
It would be better to maintain this in redux.
Anyways, with the backend you wouldnt need this id anymore.